### ISSUE TITLE
Broken links to r7rs

### ISSUE THREAD

**Issue by dradetsky** [schemeorg/community#2]
So I'm assuming this is the source repo for e.g. https://standards.scheme.org/. If not, feel free to close this issue and call me an idiot (or other mean names according to preference). Although FWIW consider a repo readme so the next guy can figure out what he's looking at, and also maybe a more prominent link from scheme.org back to this repo.

Anyhow, I went to look up r7rs to figure something out, and the link was broken. More specifically, the link labelled `Unofficial documents with errata corrected (browse on the web):` which currently points to https://standards.scheme.org/unofficial/errata-corrected-r7rs/.

FWIW, I haven't touched scheme in years, so I don't really know what's going on, but I notice that r7rs was finalized, so maybe I should have known that an unofficial doc with errata would be obsolete. However, what I was actually trying to do was to click whichever link would take me to an html version of standard, since I prefer that to pdf. So while if I'm correct, it might make sense to resolve this by just getting rid of those links, I just wanted to put in a vote for hosting an html version of the standard if one is available.

**Comment by lassik** [comment]
> So I'm assuming this is the source repo for e.g. https://standards.scheme.org/.

Yes

> If not, feel free to close this issue and call me an idiot (or other mean names according to preference).

You're not an idiot.

> Although FWIW consider a repo readme so the next guy can figure out what he's looking at, and also maybe a more prominent link from scheme.org back to this repo.

Yes. To lower the barrier to entry, we should put a link to the git repo from every page on the live site.

> Anyhow, I went to look up r7rs to figure something out, and the link was broken. More specifically, the link labelled `Unofficial documents with errata corrected (browse on the web):` which currently points to https://standards.scheme.org/unofficial/errata-corrected-r7rs/.

> FWIW, I haven't touched scheme in years, so I don't really know what's going on, but I notice that r7rs was finalized, so maybe I should have known that an unofficial doc with errata would be obsolete. However, what I was actually trying to do was to click whichever link would take me to an html version of standard, since I prefer that to pdf. So while if I'm correct, it might make sense to resolve this by just getting rid of those links, I just wanted to put in a vote for hosting an html version of the standard if one is available.

Ah, we don't have a HTML version of R7RS at all. That was wishful thinking. I'll remove the link.

The "Small Edition" of R7RS was finished in 2013. That document specifies a language slightly bigger than R5RS.

The "Large Edition" of R7RS is still work in progress. That's meant to be a superset of R7RS-small, incorporating most or all R6RS features and adding many new libraries.

Most of the time, people should read the errata-corrected documents, since as the name says, they fix mistakes in the spec. But we have to say that those documents are "unofficial" to preserve trust in the RnRS standardization process, which says that the editors of a standard approve one final version of the standard and are then free to move on to other things with no more obligations. Officially ratifying all errata corrections to the standards would be too much work for volunteers.

So, TL;DR you should probably use the [errata-corrected R7RS PDF](https://standards.scheme.org/unofficial/errata-corrected-r7rs.pdf).

**Comment by lassik** [comment]
> I just wanted to put in a vote for hosting an html version of the standard if one is available.

We'll gladly host one, but converting the TeX sources to HTML is a non-trivial project. It was done for R6RS, but not (yet) for R7RS.

**Comment by lassik** [comment]
Your wish for HTML R7RS [has been fulflled]( https://standards.scheme.org/corrected-r7rs/r7rs.html).
